components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1502,6 +1509,70 @@ func (p *pg) CreateProject(ctx context.Context, project *v2.Project) (*v2.Projec
 	return project, nil
 }
 
+func (p *pg) addSupportPolicies(ctx context.Context, project *v2.Project, q Querier) error {
+	policyParams := []struct {
+		id   string
+		name string
+		role string
+	}{
+		{
+			id:   fmt.Sprintf("%s-%s", project.ID, "project-owners"),
+			name: fmt.Sprintf("%s %s", project.Name, "Project Owners"),
```

Comment:
This is a UX call, but I think the optional upper vs lowercasing might make these names look funny sometimes